tests/integration-test/bpf/tc_pass.bpf.c

```diff
@@ -0,0 +1,15 @@
+#include <linux/bpf.h>
+#include <linux/pkt_cls.h>
+#include <bpf/bpf_helpers.h>
+
+volatile const __u32 GLOBAL_1 = 0;
+volatile const __u32 GLOBAL_2 = 0;
```

Comment:
Should we make one of these be a u8 just for a little bit of extra verification that the global data feature is functioning correctly?

Reply:
Yes, I agree.  I'm working on the e2e testing and was also thinking that u8 would be nice because the endianness wouldn't matter.

Reply:
Done